README.Rmd

```diff
@@ -15,13 +15,16 @@ knitr::opts_chunk$set(
   comment = "#>",
   fig.path = "man/figures/README-"
 )
-ver <- desc::desc_get_version(".")
-ver <- paste0("https://img.shields.io/badge/Version-", ver,
+dcf <- read.dcf(file = file.path(".", "DESCRIPTION"))
+if ( nrow(dcf) < 1L) 
+  stop("DESCRIPTION file of package is corrupt.", call. = FALSE) 
+desc <- as.list(dcf[1L, ])
+ver <- paste0("https://img.shields.io/badge/Version-", desc$Version,
               "-success.svg?style=flat&logo=github")
 ```
 
 
-# SomaDataIO <a href="https://somalogic.github.io/SomaDataIO"><img src="man/figures/logo.png" align="right" height="138" alt="" /></a>
```

Comment:
I noticed that the README title of SomaPlotr is in code-case (e.g. `SomaPlotr`). Is that something you would like to do across all the GitHub packages, or is that an accident in SomaPlotr?

Reply:
I think I'm missing something ... what's the difference? It looks right to me :handsup:

Reply:
Oh, wait, I see now ... yes, lets remove the code-case for consistency (also with `tidyverse`)